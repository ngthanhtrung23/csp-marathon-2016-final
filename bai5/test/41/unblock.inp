....1.
222314
005314
6.5XX4
6ZZR..
...R..
