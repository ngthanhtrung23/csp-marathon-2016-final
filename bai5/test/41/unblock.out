31
3 U 1
4 U 1
6 D 1
X R 1
R U 2
Z R 3
5 D 2
R D 2
X L 4
5 U 2
R U 2
Z L 3
1 D 3
4 D 3
R D 2
3 D 2
2 R 3
5 U 2
0 R 1
X R 1
6 U 4
0 L 1
X L 1
Z L 1
5 D 4
0 R 1
X R 1
6 D 2
2 L 3
3 U 2
0 R 3

